### ISSUE TITLE
Crash ID: 9a47a3c2dc341b4b47cb8a8de4b0c07eff3d853e    when opening a particular project

### ISSUE THREAD

**Issue by Simone-Brotto** [qgis/QGIS#48335]
### What is the bug or the crash?

This crash occours only when I try to open a particular project last time saved 15.03.2022

Crash ID: 9a47a3c2dc341b4b47cb8a8de4b0c07eff3d853e

Stack Trace

QgsFields::QgsFields :
QgsVectorLayer::fields :
QgsFieldFormatter::representValue :
QgsLayoutItemAttributeTable::getTableContents :
QgsLayoutItemAttributeTable::atlasLayerChanged :
QObject::qt_static_metacall :
QgsLayoutReportContext::layerChanged :
QgsLayoutReportContext::setLayer :
QgsLayoutAtlas::readXml :
QgsPrintLayout::readXml :
QgsLayoutManager::readXml :
QgsProject::readProjectFile :
QgsProject::unzip :
QgsProject::read :
QgisApp::addProject :
QgisApp::fileOpen :
QObject::qt_static_metacall :
QAction::activate :
QAbstractButton::click :
QAbstractButton::mouseReleaseEvent :
QToolButton::mouseReleaseEvent :
QWidget::event :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QApplicationPrivate::sendMouseEvent :
QSizePolicy::QSizePolicy :
QSizePolicy::QSizePolicy :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QGuiApplicationPrivate::processMouseEvent :
QWindowSystemInterface::sendWindowSystemEvents :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QEventLoop::exec :
QCoreApplication::exec :
main :
BaseThreadInitThunk :
RtlUserThreadStart :

QGIS Info
QGIS Version: 3.22.6-Bia?owie?a
QGIS code revision: b6f33b1a
Compiled against Qt: 5.15.2
Running against Qt: 5.15.2
Compiled against GDAL: 3.4.2
Running against GDAL: 3.4.2

System Info
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.19044

### Steps to reproduce the issue

1. launch QGIS 3.22
2. open particular project
3. crash

### Versions

3.22.6
3.24.2

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [X] I tried with a new QGIS profile

**Comment by nicogodet** [comment]
Can you share this project ?

**Comment by Simone-Brotto** [comment]
Here is qgz file, I can not upload all vector and raster because it is about 10 GB.
I hope it is fine. Thank you 

[QGIS_TAMER.zip](https://github.com/qgis/QGIS/files/8548196/QGIS_TAMER.zip)

**Comment by nicogodet** [comment]
No crash at opening.
But QGIS hangs (didn't wait for the complete crash) when trying to open "x" layout (could be linked to missing layers)...

**Comment by Simone-Brotto** [comment]
How can I fix the file?

**Comment by Simone-Brotto** [comment]
Here is how I solved the problem.
I copied *.qgz file into another directory without all layers, opened with QGIS 3.24.2., selected keep missing layers.
Since @nicogodet suggested it was a layout issue I tried deleting "x" layout but it was not enough, so I ended up deleting all layouts. Saved, moved to original directory, file is fixed and works fine, I lost all layouts but I can recreate them. I hope this procedure can help others. Take care. Simone Brotto
